test/hotspot/jtreg/compiler/lib/ir_framework/IRNode.java

```diff
@@ -1164,11 +1164,21 @@ public class IRNode {
         vectorNode(COUNTTRAILINGZEROS_VL, "CountTrailingZerosV", TYPE_LONG);
     }
 
+    public static final String COUNTTRAILINGZEROS_VI = VECTOR_PREFIX + "COUNTTRAILINGZEROS_VI" + POSTFIX;
+    static {
+        vectorNode(COUNTTRAILINGZEROS_VI, "CountTrailingZerosV", TYPE_INT);
+    }
+
     public static final String COUNTLEADINGZEROS_VL = VECTOR_PREFIX + "COUNTLEADINGZEROS_VL" + POSTFIX;
     static {
         vectorNode(COUNTLEADINGZEROS_VL, "CountLeadingZerosV", TYPE_LONG);
     }
 
+    public static final String COUNTLEADINGZEROS_VI = VECTOR_PREFIX + "COUNTLEADINGZEROS_VI" + POSTFIX;
```

Comment:
Would have been better to add `_` like that: `COUNT_LEADING_ZEROS_VI`

But the existing `IRNode` strings for the long versions already miss that. If you want to also fix this here, feel free to do so.

Reply:
Yes, it's more readable. Fixed.
Thanks for your reviewing.